Makefile

```diff
@@ -1,6 +1,7 @@
-# Go parameters
-GOCMD?=go
-GOENV=GO111MODULE=on GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+export GO111MODULE = on
+
+GOCMD=go
```

Comment:
We should keep the `?` so that it's overridable from the command line (we often swap out to `richgo`)

Reply:
Sorry, yep, keep it.